lib/bloc/payment_options/payment_options_bloc.dart

```diff
@@ -56,10 +66,13 @@ class PaymentOptionsBloc extends Cubit<PaymentOptionsState> {
     _log.v("Fetching payments override settings");
     final enabled = await _preferences.getPaymentOptionsOverrideFeeEnabled();
     final proportional = await _preferences.getPaymentOptionsProportionalFee();
-    _log.v("Payments override fetched: enabled: $enabled, proportional: $proportional");
+    final excemptFeeMsat = await _preferences.getPaymentOptionsExemptFee();
+    _log.v(
+        "Payments override fetched: enabled: $enabled, proportional: $proportional, excemptFeeMsat: $excemptFeeMsat");
```

Comment:
Typo in `excemptFeeMsat`
should be `exemptFeeMsat`

Reply:
Fixed 😃

Reply:
@ubbabeck there's one more thing remaining, the file name :)
`lib/routes/payment_options/widget/excempt_fee_widget.dart`

Reply:
For some reason this word tripped me up big time, believe all typos have been removed now. Except the remote branch name...